controllers/volumereplicationgroup_controller.go

```diff
@@ -610,6 +610,15 @@ func (v *VRGInstance) clusterDataRestore(result *ctrl.Result) error {
 
 		if clusterDataReady.Status == metav1.ConditionTrue &&
 			clusterDataReady.ObservedGeneration == v.instance.Generation {
+
+			if ok, err := v.allS3StoresHavePVCPrefix(); !ok || err != nil {
```

Comment:
I think we don't need to do anything for the PVCs download. Only for the upload.